setup.py

```diff
@@ -52,7 +52,7 @@ def run(self):
 setup(
     name="pocketlint",
     description="Pocket-lint a composite linter and style checker.",
-    version="1.4.4",
+    version="1.4.4.c13",
```

Comment:
why add .c13 here.

there is a bug here as `name` should be `scame` and not pocketlint

you might be working on an old fork